testing/vcs/CMakeLists.txt

```diff
@@ -978,33 +978,34 @@ cdat_add_test(test_vcs_mercator_edge
 )
 
 # Rename baseline
-cdat_add_test(test_vcs_continents
-  "${PYTHON_EXECUTABLE}"
-  ${cdat_SOURCE_DIR}/testing/vcs/test_vcs_continents.py
-  ${BASELINE_DIR}/test_vcs_continents.png
-)
+#cdat_add_test(test_vcs_continents
```

Comment:
> because they do not work on every machine? And like that it will look cleaner to Dean.

I am fine with it if could you create an issue for these (I believe I have the issue created for continents but not sure about the other test).